temba/locations/migrations/0003_adminboundary_in_country.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('locations', '0002_auto_20141126_2054'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='adminboundary',
+            name='in_country',
+            field=models.CharField(
+                help_text=b"The OSM id of this admin level's country id", max_length=15, null=True),
+            preserve_default=True,
```

Comment:
Isn't MPTT's reason for being exactly for this purpose? (to make querying in trees fast/easy?)

As I said, hate to double encode country information (both through the implicit tree and through a variable) so would be great if we could lean on MPTT for this instead of introducing the ability for us to have corrupted data.